<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="test_eof_exception_bytes.py" name="test_eof_exception_bytes.TestEofExceptionBytes-20200731203713" skipped="0" tests="1" time="0.000" timestamp="2020-07-31T20:37:13">
	<testcase classname="test_eof_exception_bytes.TestEofExceptionBytes" file="spec/python/test_eof_exception_bytes.py" line="9" name="test_eof_exception_bytes" time="0.000" timestamp="2020-07-31T20:37:13">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
